posthog/models/filters/mixins/utils.py

```diff
@@ -15,6 +15,11 @@ def include_dict(f):
     return f
 
 
+def include_query_tags(f):
```

Comment:
Nit picking but someone using this decorator also has to update the create_metrics_query_log to use it.

A comment here to call that out might help the future traveller

Reply:
Good suggestion - this is a really nice seam to document the system with.